app/tests/integration/nimbus/remote_settings/pages/dashboard.py

```diff
@@ -1,97 +1,37 @@
-from pypom import Page, Region
-from selenium.common.exceptions import NoSuchElementException
+from nimbus.pages.base import Base
```

Comment:
Why are you importing the nimbus base page here? I think, if anything, we should create a separate kinto base page.

Reply:
Oh it was to reuse the same base logic with the timeouts and the waits and stuff.  Is there something we need to put in a separate base?

Reply:
Since remote settings is _technically_ different than nimbus, I would rather have remote settings stuff be agnostic to nimbus.

Reply:
I see your point conceptually, we could have a separate base but then we'd either repeat the same stuff we put in the nimbus base, or inherit from it and there wouldn't be anything specific to remote settings in it.  I generally don't add a class until it's necessary, and if/when there's something specific to remote settings we could add its own base class but I don't think there is anything yet?

Reply:
We could also move the nimbus pages into a sub folder like we did with remote settings and leave base at the root so it's clear structurally that it's a shared base?

Reply:
There isn't anything yet that is specific to remote settings but there are things in that base class that don't belong to remote settings. It's not a huge deal but it will bother me 😁

Reply:
Ya I think if we do it the way I'm suggesting we move the experimenter specific stuff out, one sec....

Reply:
Actually there was something to put in a remote settings base!  So now there's Base (timeouts/waits), ExperimenterBase (all the save stuff), RemoteSettingsBase (with a shared locator!)

Reply:
Much nicer 👍